frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -6,17 +6,25 @@ import SideBar from "../components/SideBar";
 const MonthlyWorkingHours = () => {
   const [data, setData] = useState([]);
   const [selectedYear, setSelectedYear] = useState(2024);
+  const [selectedAmount, setSelectedAmount] = useState(3);
 
   const onYearChange = (e) => {
     setSelectedYear(e.target.value);
   };
 
+  const onAmountChange = (value) => {
+    setSelectedAmount(value);
+  };
+
   const applyFilters = () => {
```

Comment:
This function just calls the getYears function. It would be better to avoid functions that do the same but have a different name. Could you remove this function and just call getYears wherever it needs to be called?